tests/run_vector_tests.sh

```diff
@@ -28,7 +28,7 @@ HARDWARE_ID=test_primary_hardware_id
 
 "$TTV_DIR/generator.py" --signature-encoding base64 -o vectors --cjson json-subset \
                         --ecu-identifier $ECU_SERIAL --hardware-id $HARDWARE_ID
-# disable werkzeug debug pin which causes issues on Jenkins
+# disable werkzeug debug pin which caused issues on Jenkins
```

Comment:
Is this still necessary? It's fine if you want to leave it, just checking.

Reply:
Ok, opened https://github.com/advancedtelematic/tuf-test-vectors/pull/55